.cargo/config.toml

```diff
@@ -0,0 +1,7 @@
+# Custom getrandom backend for RISC Zero zkVM target
+[target.riscv32im-risc0-zkvm-elf]
+rustflags = ['--cfg', 'getrandom_backend="custom"']
+
+# Custom getrandom backend for SP1 zkVM target
+[target.riscv32im-succinct-zkvm-elf]
+rustflags = ['--cfg', 'getrandom_backend="custom"']
```

Comment:
you should do the suggestion by @syjn99 we want to support all zkVM's, not target them individually